src/poliastro/tests/test_plotting2d.py

```diff
@@ -92,7 +96,10 @@ def test_savefig_calls_prepare_plot():
     m = OrbitPlotter2D()
     earth = Orbit.from_body_ephem(Earth)
     m.plot(orbit=earth, label="Obj")
+    pwd = os.getcwd()
```

Comment:
Could you use

```
with tempfile.NamedTemporaryFile(...) as fp:
    m.savefig(filename=fp.name)
```

instead? I think it's much clearer

Reply:
Oh yes! 
And sorry I didn't know that the cell was on purpose :sweat_smile:

Reply:
Should I add a markup cell explaining the same thing? That the cell is on purpose? :sweat: